pkgs/top-level/all-packages.nix

```diff
@@ -28694,6 +28705,12 @@ with pkgs;
 
   owamp = callPackage ../applications/networking/owamp { };
 
+  vgmplay-libvgm = callPackage ../applications/audio/vgmplay-libvgm {
+    inherit (darwin) libiconv;
```

Comment:
There is a toplevel libiconv which points to the correct version for linux and darwin. I think we should use that one instead.

Reply:
Done.